### ISSUE TITLE
Add role="switch" to switches in example code?

### ISSUE THREAD

**Issue by coliff** [twbs/bootstrap#34818]
I’ve just been reading this good article by @argyleink about building a switch component
https://web.dev/building-a-switch-component/#overview

I see it uses `role=switch`. Think this should be added to the Switch Bootstrap docs example code?
https://getbootstrap.com/docs/5.1/forms/checks-radios/#switches

Let me know, I’d be happy to open a PR for it.

**Comment by ffoodd** [comment]
I think it's been discussed already, but can't find where. Friendly ping @patrickhlauke, IMHO it only concerns accessibility 🙂
